sqlalchemy-stubs/ext/asyncio/base.pyi

```diff
@@ -1,17 +1,23 @@
 import abc
 from typing import Any
+from typing import Generator
+from typing import TypeVar
+
+_TStartableContext = TypeVar("_TStartableContext", bound=StartableContext)
```

Comment:
in pyi files values can be referenced before definition?

Reply:
Yes